chapters/unitexpressions.tex

```diff
@@ -42,11 +42,13 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
   unit-operand [ unit-exponent ]
 
 unit-exponent :
-   [ "+" | "-" ] UNSIGNED-INTEGER
+   [ "+" | "-" ] ( UNSIGNED-INTEGER | "(" UNSIGNED-INTEGER "/" UNSIGNED-INTEGER ")" )
 \end{lstlisting}
 
 The SI standard uses super-script for the exponentation, and does thus not define any operator symbol for exponentiation.
-A \lstinline[language=grammar]!unit-factor! consists of a \lstinline[language=grammar]!unit-operand! possibly suffixed by a possibly signed integer number, which is interpreted as an exponent.
+A \lstinline[language=grammar]!unit-factor! normally consists of a \lstinline[language=grammar]!unit-operand! possibly suffixed by a possibly signed integer number, which is interpreted as an exponent.
+Rational exponents are needed to fully support the unit for square roots (and can be used for roots in general).
+They are intended to be only be used when needed, i.e., not to represent integer exponents.
 There must be no spacing between the \lstinline[language=grammar]!unit-operand! and a possible \lstinline[language=grammar]!unit-exponent!.
```

Comment:
Better to keep the recommendation separate from the more normative description:
```suggestion
There must be no spacing between the \lstinline[language=grammar]!unit-operand! and a possible \lstinline[language=grammar]!unit-exponent!.
It is recommended to use the simplest representation of exponents, meaning that the explicit \lstinline!+! sign should be avoided, that leading zeros should be avoided, that rational exponents are reduced to not have common factors in the numerator and denominator, and that rational exponents with denominator 1 should be avoided in favor of integer exponents.

\begin{nonnormative}
Applications of rational exponents include units of square roots, units inferred from polynomial equations, and units of quantities used in certain modeling domains.
\end{nonnormative}
```

Reply:
I must say I'm skeptical about including the non-normative section at all.  I'd like to avoid a situation where we build up the expectation that every part of the language comes with a non-normative motivation.  It would at least good to have a quick language group poll about including it.

Reply:
Since the development is in git I think we can do the change above, but remove the non-normative part.

Reply:
And possibly add the non-normative part if needed.